Zenodo.js

```diff
@@ -175,6 +175,7 @@ function scrape(doc, url) {
 				if (!item.creators[i].firstName && item.creators[i].lastName.indexOf(",")!=-1) {
 					item.creators[i].firstName = item.creators[i].lastName.replace(/.+?,\s*/, "");
 					item.creators[i].lastName = item.creators[i].lastName.replace(/,.+/, "");
+					item.creators[i].fieldMode = true;
 				}
```

Comment:
Does this work? No changes in the test cases??

I think what we want is rather want is:
```js
if (!item.creators[i].firstName) {
	if (item.creators[i].lastName.indexOf(",")!=-1) {
		item.creators[i].firstName = item.creators[i].lastName.replace(/.+?,\s*/, "");
		item.creators[i].lastName = item.creators[i].lastName.replace(/,.+/, "");
	} else {
		item.creators[i].fieldMode = true;
	}
}
```